internal/flycheck/pg.go

```diff
@@ -55,6 +59,14 @@ func CheckPostgreSQL(ctx context.Context, checks *check.CheckSuite) (*check.Chec
 	return checks, nil
 }
 
+func repmgrCheck() (string, error) {
+	cmd, err := utils.RunCommand("repmgr node check --config-file /data/repmgr.conf", "postgres")
```

Comment:
We should pull the `--config-file` value from `node.RepMgr.ConfigPath` instead of hard-coding it.

Reply:
nit: `cmd` seems like a weird variable name. :D

Reply:
> We don't have a great way to access this from the healthcheck server
@DAlperin is this blocking?